src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -104,18 +99,27 @@ public void editPerson_dataValidation() {
 
     @Test
     public void cancelOperation_usingAccelerator() {
+        Person newAlice = new PersonBuilder(td.alice.copy()).withFirstName("Alicia").withLastName("Brownstone")
+                .withStreet("Updated street").withCity("Singapore").withPostalCode("123123")
+                .withBirthday("01.01.1979").withGithubUsername("alicebrown123").withTags(td.colleagues, td.friends).build();
```

Comment:
I know you are following my code, but on retrospect, this declaration should be done just before the newAlice is used. In general, declare variables close to where they are used first.